crates/iota-sdk-ffi/src/types/signature.rs

```diff
@@ -93,6 +95,13 @@ impl UserSignature {
         ))
     }
 
+    #[uniffi::constructor]
+    pub fn new_move(authenticator: &MoveAuthenticator) -> Self {
```

Comment:
Please add `_authenticator` suffix to this and passkey/zklogin etc. I'd be fine with `_auth` as well

Reply:
okay I did them all